File: Shovel.cpp
#include <bits/stdc++.h>
using namespace std;

int main(void) {
    //variable declaration and input
    int price, denomination;
    cin >> price >> denomination;
    
    // main logic
    for (int x=1; x<10; x++) {
        // got this from tutorial
        if (price*x%10==0 || price*x%10==denomination) {
            cout << x << endl;
            break;
        }
    }

    return 0;
}

File: Python/Shovel.py
def main():
    price = int(input())
    denomination = int(input())
    for x in range(1, 10):
        if price*x % 10 == 0 or price*x % 10 == denomination:
            print(x)
            break


if __name__ == '__main__':
    main()
